examples/sdk/node/src/index.ts

```diff
@@ -63,48 +63,68 @@ function sendMetrics() {
     }
     client.metrics.send();
 }
+
+function oom() {
+    function allocateMemory(size: number) {
+        const numbers = size / 8;
+        const arr = [];
+        arr.length = numbers;
+        for (let i = 0; i < numbers; i++) {
+            arr[i] = i;
+        }
+        return arr;
+    }
+
+    const TIME_INTERVAL_IN_MSEC = 40;
+    const memoryLeakAllocations = [];
+
+    console.log('This may take a while dependning on Node memory limits.');
+    console.log('For best results, start with --max-old-space-size set to a low value, like 100.');
+    console.log('e.g. node --max-old-space-size=100 lib/index.js');
+    setInterval(() => {
+        const allocation = allocateMemory(10 * 1024 * 1024);
+        memoryLeakAllocations.push(allocation);
+    }, TIME_INTERVAL_IN_MSEC);
+
+    return new Promise(() => {
+        // Never resolve
+    });
+}
+
 function showMenu() {
+    const options = [
+        ['Send an exception', (attributes: Record<string, number>) => sendHandledException(attributes)],
+        ['Send a message', (attributes: Record<string, number>) => sendMessage('test message', attributes)],
+        ['Throw rejected promise', () => rejectPromise('Rejected promise')],
+        ['Add a new summed event', (attributes: Record<string, number>) => addEvent('Option clicked', attributes)],
+        ['Send all metrics', sendMetrics],
+        ['OOM', oom],
```

Comment:
What do you think about moving OOM just below rejected promise? For me it makes more sense because we're grouping actions one by one

Reply:
done